From 7b654cae60746260fc63cf29b8e1cc589e734513 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Mon, 28 Mar 2016 13:36:14 +0800
Subject: [PATCH 5/5] gettext-runtime/intl/printf-parse.c: Fix build on Visual
 Studio 2008

We don't have stdint.h nor inttypes.h shipped with Visual Studio 2008, so
make sure we define intmax_t accordingly.
---
 gettext-runtime/intl/printf-parse.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gettext-runtime/intl/printf-parse.c b/gettext-runtime/intl/printf-parse.c
index 87f49a3..b0cc1b1 100644
--- a/gettext-runtime/intl/printf-parse.c
+++ b/gettext-runtime/intl/printf-parse.c
@@ -56,6 +56,9 @@
 # if HAVE_INTTYPES_H_WITH_UINTMAX
 #  include <inttypes.h>
 # endif
+# if !defined (HAVE_STDINT_H_WITH_UINTMAX) && !defined (HAVE_INTTYPES_H_WITH_UINTMAX) && defined (_WIN32)
+#  define intmax_t long long
+# endif
 #else
 # include <stdint.h>
 #endif
-- 
2.7.1.windows.2

